### DISCUSSION TITLE
When A New 3D (Stereoscopie) Channel?

### DISCUSSION THREAD

**Discussion by cheznewa** [iptv-org/community#282] [Q&A] [answered]
A Channel 3D (Stereoscopie) Will Identified If The Two Videos Left/Right Half Side By Side.
When There A Channels 3D?

**Comment by freearhey** [comment, marked as answer]
There are already channels offering 3D content. Some of them are listed here: https://en.wikipedia.org/wiki/3D_television#3D_channels
